test.sh.in

```diff
@@ -448,19 +573,12 @@ if [ "$RUN_CIT_TEST" == true ] || [ "$RUN_FIT_TEST" == true ] ; then
     # time to make the env instead of doing sleeps...
     setupVirtualEnv
     waitForAPI
-    nodesOn &
+    nodesOn
+
     generateSolLog
     vnc_record_start
     # Run tests
-    runTests
-    generateSysLog
-    generateMongoLog
-    vnc_record_stop
-    # Clean Up below
-
-    #shutdown vagrant box and delete all resource (like removing vm disk files in "~/VirtualBox VMs/")
-    #cleanupVMs
-    #nodesDelete
+    runTestsForDocker
```

Comment:
Yes, but we will do it in jobs/FunctionTest/cleanup.sh
Because if there are failed cases, we need to save the container as a image and push it to docker hub.